README.md

```diff
@@ -14,6 +14,15 @@ $ tns plugin add nativescript-iqkeyboardmanager
 
 That's it! IQKeyboardManager takes care of all initialization when your app starts up by default.
 
+## Advanced usage
+
+Want to tweak the appearance and/or behavior?
+
+Take a look at the embedded [demo app](demo/) for a bunch of features.
+You can easily access [any other feature](https://github.com/hackiftekhar/IQKeyboardManager/blob/df3afc60ad92a2264dc5e101a3c7966d7f6265ef/PROPERTIES%20%26%20FUNCTIONS.md) because this plugin exposes the entire library through [a TypeScript definition file](demo/references.d.ts).
```

Comment:
@EddyVerbruggen Ah, by “just this” I meant doing to a link to the demo app’s `tsconfig.json` file after we update it. Here’s the [`tsconfig.json` from Groceries](https://github.com/NativeScript/sample-Groceries/blob/master/tsconfig.json). The {N} CLI generates this file for you now so that demo app should have the config in place—we should just have to add the path to this package’s `.d.ts` file.